build/ci-build.yml

```diff
@@ -67,18 +75,10 @@ stages:
             inputs:
               artifact: 'Build'
               path: '$(Build.SourcesDirectory)'
-          - task: DotNetCoreInstaller@0
-            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
-            inputs:
-              version: '$(DotNet.Sdk.Version)'
-          - task: DotNetCoreCLI@2
-            displayName: 'Run unit tests'
-            inputs:
-              command: test
-              projects: 'src/**/$(Project).Tests.Unit.csproj'
-              arguments: '--configuration $(Build.Configuration)'
```

Comment:
Ha no version prefix.

What would you do @stijnmoreels : Do we add something fixed and don't care or see if we can add ifs?

Reply:
Version of the build nuget package? Or from the .net SDK? I think you talk about the variable that has 'preview' as value?

Reply:
Other builds use --version-prefix, which is now part of the task but not the CI build.

Reply:
Yeah, could we have optional parameters in the template? So we can ignore the version prefix in the CI build?

Reply:
We could, we'd just need to figure out how it works in terms of passing the parameter or not.

Should have some time later on to figure that out unless you can already do some research on this ofcourse.

Reply:
Resolving as we're tracking this in https://github.com/arcus-azure/azure-devops-templates/issues/7